demos/tutorials/fun_vec_representation.py

```diff
@@ -0,0 +1,266 @@
+"""
```

Comment:
Thank you for this very helpful tutorial. Overall I think the flow looks good of the funvals, vector etc.

However, I would like to have some more realistic examples in this tutorial so that it is easier to reason about and understand.

My suggestion is to show-case at least these two examples in the tutorial.
1) A 2D geometry with an image taken e.g. from `cuqi.data` such as grains, threephases or astronaut. Then a mapping that e.g. does a thresholding which is clearly visible compared to the normal image. E.g. put all values below 100 to 0 or likewise.
2) A 1D geometry such as KL Expansion or StepExpansion to illustrate difference between funvals and parameters.

I believe you should be able to take the existing tutorial and put in the more realistic examples.

Reply:
Thank you for the suggestion! I updated the tutorial using KLExpantion and a simple 2D smile image. I think these updates clarifies the funval conversion in a much better way than the original examples in the tutorial 👍 . Let me know what you think.